.travis.yml

```diff
@@ -0,0 +1,12 @@
+
+language: python
+
+python:
+  - "3.5"
+
+install:
+  - pip install -r requirements.txt
```

Comment:
You might want to add `flake8` to your `requirements.txt` so the below script actually works. :slightly_smiling_face:

Reply:
@varunagrawal Added! :+1: